eos-phone-home

```diff
@@ -149,11 +151,10 @@ class PhoneHome(object):
                             ['attr', '-q', '-g', 'eos-image-version', path])
                 image = image.decode(errors='replace').strip()
 
-                print(" - Image from %s: %s" % (path, image))
+                log.info(" - Image from %s: %s", path, image)
                 return image
-            except Exception as e:
-                print("Unable to get %s image name!" % (path, ), e,
-                      file=sys.stderr)
+            except subprocess.CalledProcessError:
+                log.info("Unable to get image name from %s", path)
```

Comment:
I don't think the exception is valuable. Here's the output without, as in this branch (I'm running on my Fedora system so neither attr exists):

```
INFO:__main__:Getting variable: image
attr_get: No such file or directory
Could not get "eos-image-version" for /sysroot
INFO:__main__:Unable to get image name from /sysroot
attr_get: No data available
Could not get "eos-image-version" for /
INFO:__main__:Unable to get image name from /
```

And with:

```
INFO:__main__:Getting variable: image
attr_get: No such file or directory
Could not get "eos-image-version" for /sysroot
INFO:__main__:Unable to get image name from /sysroot
Traceback (most recent call last):
  File "./eos-phone-home", line 151, in _get_image
    ['attr', '-q', '-g', 'eos-image-version', path])
  File "/usr/lib64/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/lib64/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['attr', '-q', '-g', 'eos-image-version', '/sysroot']' returned non-zero exit status 1.
attr_get: No data available
Could not get "eos-image-version" for /
INFO:__main__:Unable to get image name from /
Traceback (most recent call last):
  File "./eos-phone-home", line 151, in _get_image
    ['attr', '-q', '-g', 'eos-image-version', path])
  File "/usr/lib64/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/lib64/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['attr', '-q', '-g', 'eos-image-version', '/']' returned non-zero exit status 1.
```

The interesting error is in the called process's `stderr` output, not in the exception. We know the exit code is non-zero, and the backtrace is not interesting – one or other call is expected to fail depending on whether you're on a converted or unconverted system.